### ISSUE TITLE
music packs

### ISSUE THREAD

**Issue by AppaBTW** [Sequitur-Studios/Cell-Machine-Mystic-Mod#39]
a way to custom music into the game like texture packs but for music, I think it will be a great idea since it will bring more player customization and will allow the community to be more creative 
this feature could be separate to texture packs or in combination to match a theme, if someone makes a texture pack themed after a pond they could add relaxing music to go with it 
I wish to add custom music to the game and that is my only request, thank you

**Comment by jjblock21** [comment]
Would probably take long to load because sound files (aspecially the music file) are quite big.

**Comment by TheMysticLynx** [comment]
Custom music packs seems a little pointless, custom sound packs in general would make more sense. @SnazzGass Please review

**Comment by AppaBTW** [comment]
> Custom music packs seems a little pointless, custom sound packs in general would make more sense. @SnazzGass Please review

ok that is better

**Comment by SnazzGass** [comment]
I agree with Mystic. Custom sounds would be a better approach. Perhaps they could be included alongside lua mods

**Comment by BlockOG** [comment]
> I agree with Mystic. Custom sounds would be a better approach. Perhaps they could be included alongside lua mods

Why lua :(

**Comment by f1shy-dev** [comment]
Sound packs would make more sense as part of textures, which we could then change to resource packs or something similar.

**Comment by ValkyrieXD** [comment]
Great idea!
